File: LAB1/myFirstLabScript.py
def q1():
    name = input("What is your name?")
    print(f"Hello {name}")

    id = int(input("What is your Student ID?"))
    print(f"Your ID is {id}")

def q2():
    var1 = float(input("var1 : "))
    var2 = float(input("var2 : "))

    sum = var1 + var2
    diff = (var1 - var2)
    prod = var1 * var2

    print(f"Var1 {var1}\n Var2 {var2}\n Sum {sum}\n Diff {diff}\n Prod {prod} ")

def q3():
    name = str(input("What's your name: "))
    lab = int(input("Lab note: "))
    midterm = int(input("Midterm: "))
    final = int(input("final: "))

    print(f"Last Score: {lab*25/100 + midterm*35/100 + final*40/100}")

def q4():
    print("*\n**\n***\n**\n*")
def main():
    q1()
    q2()
    q3()
    q4()



main()

File: LAB1/myFirstLabScript.cpp
#include <iostream>
using namespace std;

void q1() {
    cout << "What is your name?" << endl;
    string name;
    cin >> name;
    cout << "Hello " << name << endl;
    cout << "What is your id?" << endl;
    int id;
    cin >> id;
    cout << "Your ID is " << id << endl;
}
void q2() {
    cout << "Var1" << endl;
    int var1;
    cin >> var1;
    cout << "Var2" << endl;
    int var2;
    cin >> var2;

    float sum = var1 + var2;
    cout << "Sum is " << sum << endl;
    float diff = var1 - var2;
    cout << "Diff is " << diff << endl;
    float product = var1 * var2;
    cout << "Product is " << product << endl;

}
void q3() {
    cout << "Name " << endl;
    string name;
    cin >> name;
    cout << "Lab " << endl;
    float lab;
    cin >> lab;
    cout << "Midterm " << endl;
    float midterm;
    cin >> midterm;
    cout << "Final " << endl;
    float final;
    cin >> final;
    float lastScore = lab*25/100 + midterm*35/100 + final*40/100;
    cout << "Last score is " << lastScore << endl;
}


int q4(){
    cout<<"*\n**\n***\n**\n*";
    return 0;
}

int main() {
    q1();
    q2();
    q3();
    q4();
    return 0;

}